sagemaker_batch_transform/tensorflow_open-images_tfrecord/tensorflow-serving-tfrecord-python-sdk.ipynb

```diff
@@ -61,7 +61,9 @@
   {
```

Comment:
<p>See above comments about downloaded binaries</p><p> </p><pre class="ql-syntax" spellcheck="false">!aws s3 cp s3://sagemaker-sample-data-{region}/batch-transform/open-images/tfrecord/train-00001-of-00100 /tmp/tfrecord
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1906/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>fixed for both notebooks</p>